integrationtest/couchdb/src/test/java/org/hibernate/ogm/test/integration/jboss/CouchDBModuleMemberRegistrationIT.java

```diff
@@ -77,6 +77,7 @@ private static PersistenceDescriptor persistenceXml() {
 							.createProperty().name( "hibernate.ogm.datastore.database" ).value( "ogm_test_database" ).up()
 							.createProperty().name( "hibernate.ogm.datastore.create_database" ).value( "true" ).up()
 							.createProperty().name( "hibernate.search.default.directory_provider" ).value( "ram" ).up()
+							.createProperty().name( "hibernate.transaction.jta.platform" ).value( "JBossAS" ).up()
```

Comment:
you're anticipating a draft I was writing to hibernate-dev :)

This was needed already, our documentation even mentions this requirement when running on WildFly.
So technically all I did is to align our integration tests to our docs, and doing so the JTS module was no longer needed.

I'm afraid the right platform lookup is being injected by jipijapa so not something we control, will check and then share the details ..